src/bootstrap/doc.rs

```diff
@@ -439,7 +439,7 @@ impl Step for Std {
                 builder.cargo(compiler, Mode::Std, SourceType::InTree, target, "rustdoc");
             compile::std_cargo(builder, target, compiler.stage, &mut cargo);
 
-            // Keep a whitelist so we do not build internal stdlib crates, these will be
+            // Keep an allowlist so we do not build internal stdlib crates, these will be
             // build by the rustc step later if enabled.
```

Comment:
```suggestion
            // Avoid building internal stdlib crates, these will be
            // built by the rustc step later if enabled.
```

Reply:
This comment disappeared entirely, can you reinstate it?

Reply:
Done, thanks for the suggestion.